windtunnel/windtunnel.py

```diff
@@ -103,6 +103,16 @@ def set_object(self,
         """
 
         mesh = pv.read(object_path)
+
+        if isinstance(mesh, pv.MultiBlock):
```

Comment:
a bit unclear what this is doing... is it related to the description of this pull request (save triangle meshes) or something else? Perhaps encapsulating this under a function and giving it a clear name would help...